### PR TITLE
Changing SVOX to use AOSP due to DMCA takedown

### PR DESCRIPTION
**Pull request by matthiassb**

### CODE REVIEW

**Comment by morphis** [comment]
Why was this PR closed?

**Comment by matthiassb** [comment]
Didn't know I closed this, reopening it.

File: default.xml [hunk 1]
```diff
@@ -356,4 +356,11 @@
   <project path="external/backports-wireless" name="milaq/android_external_backports-wireless" revision="3.10-2" />
   <project path="hardware/atheros/wlan" name="milaq/android_hardware_atheros_wlan" revision="master" />
   <project path="external/memnotify" name="webOS-ports/memnotify-module" revision="master" />
+  
+  <!-- jftle -->
```
Comment (morphis) [inline default.xml:360]: What does "jftle" mean? Is it the machine name?

Comment (matthiassb) [inline default.xml:360]: That commit is for the Galaxy S4 variants only the first is needed for the SVOX DCMA change. Was too lazy to rebase

**Pull request merged**